Riot/Modules/Analytics/Analytics.swift

```diff
@@ -116,7 +117,7 @@ import AnalyticsEvents
             
             switch result {
             case .success(let settings):
-                self.identify(with: settings)
+                self.identify(with: settings, and: userProperties)
```

Comment:
Just an API nitpick: here we have `identify(..., and: userProperties` and elsewhere `client.identify(..., userProperties: userProperties)`

Reply:
Good point, will rename this one to `self.identify(with: settings, userProperties: userProperties)` 👍